ui/app/templates/components/job-editor.hbs

```diff
@@ -22,10 +22,23 @@
   <header class="run-job-header">
     <h1 class="title is-3">Run a job</h1>
     <p>Paste or author HCL or JSON to submit to your cluster. A plan will be requested before the job is submitted. You can also attach a job spec by uploading a job file or dragging &amp; dropping a file to the editor.</p>
-    <label class="job-spec-upload">
-      <input type="file" onchange={{action this.uploadJobSpec}} accept=".hcl,.json,.nomad" />
-      <span class="button">Upload a job spec file</span>
-    </label>
+    <div>
+      <label class="job-spec-upload">
+        <input type="file" onchange={{action this.uploadJobSpec}} accept=".hcl,.json,.nomad" />
+        <span class="button is-outline is-small">Upload file</span>
+      </label>
+      {{#let (cannot "write variable" path="*" namespace="*") as |editVar|}}
```

Comment:
`editVar` is perhaps a misnomer; I would expect that this implies I _can_  edit the variable but it seems that just the opposite is true.

Reply:
The ticket did say, "Choose from Template requires `can edit` variable ability". So I tried to match that. What would you suggest?

Reply:
we're naming "The inability to write a variable" as "editVar", so at the very most verbose, something like "cannotEditVar" would be more correct. I think you probably just meant this to be inverted.